README.md

```diff
@@ -500,9 +500,6 @@ In such a case you can get the instance of the used wizard component in your own
 public wizard: WizardComponent;
 ```
 
-After obtaining a `WizardComponent` object, you can then obtain the `WizardState` instance of the wizard via `wizard.model` and
-the `NavigationMode` instance via `wizard.navigation`.
```

Comment:
I think it would be nice to add a short paragraph here where we describe how the wizard can be navigated from outside

Reply:
Done in e6638f0